codalab/worker_manager/slurm_batch_worker_manager.py

```diff
@@ -0,0 +1,240 @@
+import argparse
+import logging
+import uuid
+import subprocess
+import getpass
+from pathlib import Path
+import random
+import re
+
+from .worker_manager import WorkerManager, WorkerJob
+
+logger = logging.getLogger(__name__)
+
+
+class SlurmBatchWorkerManager(WorkerManager):
+    NAME = 'slurm-batch'
+    DESCRIPTION = 'Worker manager for submitting jobs using Slurm Batch'
+
+    SRUN_COMMAND = 'srun'
+    SBATCH_COMMAND = 'sbatch'
+    SBATCH_PREFIX = '#SBATCH'
+    SQUEUE_COMMAND = 'squeue'
+    SBATCH_COMMAND_RETRUN_REGEX = re.compile(r'^Submitted batch job (\d+)$')
+
+    @staticmethod
+    def add_arguments_to_subparser(subparser):
+        subparser.add_argument(
+            '--job-definition-name',
```

Comment:
Can this get changed to "job name"? Job definitions are an AWS Batch-specific concept.

Reply:
Done